src/PhpSpreadsheet/Calculation/MathTrig.php

```diff
@@ -212,47 +212,6 @@ public static function BASE($number, $radix, $minLength = null)
         return Functions::VALUE();
     }
 
-    /**
```

Comment:
Can you leave the docblock and stub code that calls these new class/methods in the MathTrig class with a @deprecated notice

[You can see my examples in the Engineering class in master branch](https://github.com/PHPOffice/PhpSpreadsheet/blob/42ecc270ecfb85ea7c419481cd3622be2e6d1e7e/src/PhpSpreadsheet/Calculation/Engineering.php#L46).

It is possible to call these MathTrig methods directly without using the calculation engine (I believe that one of our examples in sample code actually does this, or used to do so); and this allows us to flag the approach as deprecated now, so that existing code will still work; but prepares for removal in a 2.0.0 release.

Reply:
Pushed a new set of changes. All of the routines which had existed are now restored as deprecated comment block plus stubs in MathTrig. CEILINGMATH and CEILINGPRECISE are new, and I did not create deprecated stubs for them. I added a cursory test member to confirm that each of the deprecated calls continues to work as expected. That member should be deleted when the deprecated routines are removed.